employees/common/constants.py

```diff
@@ -9,3 +9,9 @@ class ReportModelConstants(Enum):
     MAX_WORK_HOURS = Decimal("24.00")
     MIN_WORK_HOURS = Decimal("0.01")
     MAX_WORK_HOURS_DECIMAL_VALUE = Decimal("0.59")
+
+
+MONTH_NAVIGATION_FORM_MAX_MONTH_VALUE = 12
+MONTH_NAVIGATION_FORM_MIN_MONTH_VALUE = 1
+MONTH_NAVIGATION_FORM_MAX_YEAR_VALUE = 2099
+MONTH_NAVIGATION_FORM_MIN_YEAR_VALUE = 1990
```

Comment:
I figured this could be the year CodePoets was founded, but I forgot to ask about it and shoot me dead guys, I couldn't remember for the life of me, which anniversary we had back in autumn (5th, I checked now...). Unless it's more sturdy to start from 2000?